mk/spksrc.spk.mk

```diff
@@ -599,11 +599,10 @@ kernel-modules-%:
 	   archs2process=$* ; \
 	fi ; \
 	$(MSG) ARCH to be processed: $${archs2process} ; \
-	set -e ; \
```

Comment:
I should explain why I'm reverting this. This is still useful when working on a package for a specific architecture. But when building dm-crypt for multiple architectures (e.g. for the aarch64 package), some of them are expected to fail because their kernel just doesn't support it. But the overall build should still succeed.